File: app/src/main/java/jp/juggler/subwaytooter/PrefDevice.java
package jp.juggler.subwaytooter;

import android.content.Context;
import android.content.SharedPreferences;

public class PrefDevice {
	
	private static String file_name = "device";
	
	static SharedPreferences prefDevice( Context context ){
		return context.getSharedPreferences( file_name, Context.MODE_PRIVATE );
	}
	
	static final String KEY_DEVICE_TOKEN = "device_token";
	static final String KEY_INSTALL_ID = "install_id";
}

File: app/src/main/java/jp/juggler/subwaytooter/PrefDevice.kt
package jp.juggler.subwaytooter

import android.content.Context
import android.content.SharedPreferences

object PrefDevice {
	
	private const val file_name = "device"
	
	fun prefDevice(context : Context) : SharedPreferences {
		return context.getSharedPreferences(file_name, Context.MODE_PRIVATE)
	}

	internal const val KEY_DEVICE_TOKEN = "device_token"
	internal const val KEY_INSTALL_ID = "install_id"
	
	const val LAST_AUTH_INSTANCE="lastAuthInstance"
	const val LAST_AUTH_SECRET="lastAuthSecret"
	const val LAST_AUTH_DB_ID ="lastAuthDbId"
}
